chainer/training/extensions/log_report.py

```diff
@@ -56,17 +56,26 @@ class LogReport(extension.Extension):
             formatting. For example, users can use '{iteration}' to separate
             the log files for different iterations. If the log name is None, it
             does not output the log to any file.
+            It is recommended to suffix the filename with ``'.jsonl'`` when
```

Comment:
A grammatical suggestion... but I also suggest either (1) write the reason of the recommendation or (2) just remove this sentence, because if this recommendation is just to make the content clear for humans, there are other cases that the output is processed just in an automated way, in which cases you don't need the suffix (e.g. processed through a pipe). If there's a recommendation without an explicit reason, one might think they would need one in those cases as well.

```suggestion
            It is recommended that you suffix the filename with ``'.jsonl'`` when
```

Reply:
This recommendation is to avoid using the same filename (`log`) in different formats because some existing third-party software (like Chainer UI) assume it has the traditional JSON array format. I'll add the reason of it.